index.html

```diff
@@ -68,6 +68,165 @@ <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
     </main>
     <!-- END -->
 
+    <!-- WORKS SECTION -->
+    <section id="portfolio" class="work__section">
+        <article class="work__description">
+            <figure class="work__image__container">
+                <img src="/assets/wrok1.png" alt="first work image" class="work__image">
```

Comment:
- [ ] Kindly, make sure the images display correctly on the browser. The path for image files seems to be broker (or at least this format may not work on all device, mine included). Consider updating the path from `"/assets/wrok1.png"` to `./assets/wrok1.png` to fix the issue. Please do the same in all other images of the project.

This will make your project way prettier and good looking 😄